homeassistant/components/ecovacs/vacuum.py

```diff
@@ -319,6 +324,10 @@ def extra_state_attributes(self) -> Mapping[str, Any] | None:
 
     async def async_set_fan_speed(self, fan_speed: str, **kwargs: Any) -> None:
         """Set fan speed."""
+        if TYPE_CHECKING:
+            # mypy does not understand that VacuumEntityFeature.FAN_SPEED is only set
+            # if self._capability.fan_speed is not None (see __init__)
```

Comment:
But mypy doesn't look at the Vacuum entity features, so this comment is a bit strange and I think the code is self-explanatory